### DISCUSSION TITLE
Question about github health files licensing

### DISCUSSION THREAD

**Discussion by nnmrts** [unifiedjs/unified#129] [Q&A] [answered]
I'd like to use unified's [community health files](https://github.com/unifiedjs/.github), especially the [support.md](https://github.com/unifiedjs/.github/blob/main/support.md) as templates for my own projects, but I'm unsure how to appropriately give credit to @wooorm according to the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) license.

Is it enough to put the repository or @wooorm in the "Acknowledgments" section of my readme?
Since I probably am gonna change a little bit in the support.md, am I allowed to replace the copyright notice at the bottom with my own one, as long as I'm also giving credit to @wooorm for the initial file?

I realize this is a general question about CC-BY-4.0, but in this specific case of "just" basically copying a markdown document, I'm really unsure how to implement a proper attribution.

**Comment by wooorm** [comment, marked as answer]
The question boils down to “what is a work?” Are you making something new? Is it a derivative? The answer boils down to where you are located, and a lot of other complex stuff, without an easy answer.

You are allowed to use it, indeed. So long as you give appropriate credit. CC gives more examples of how to dot that.
The thing IMO is that you a) note my name and the copyright (locking down all rights), b) note the license (giving rights back).

**Reply by wooorm** [reply]
The closest answer I can give is: a) add a link and such to the acknowledgements, b) the license is CC-BY-4.0 © Your Name, Titus Wormer
